README.md

```diff
@@ -7,22 +7,68 @@
   Innovaccer Design System
 </h1>
 <h3 align="center">
-  A React based UI Toolkit for building Web Applications.
+  A React-based UI toolkit for building Web Applications.
 </h3>
 
 
 ## 🚀 Get Up and Running
 
+There are two main steps to use a component in your project installing it with npm or yarn and importing stylesheet.
```

Comment:
There are two main steps to use a component in your project:
- Installing component library using npm or yarn
- Importing stylesheet